flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/table/JdbcDynamicTableFactoryTest.java

```diff
@@ -144,7 +146,8 @@ void testJdbcReadProperties() {
                         readOptions,
                         LookupOptions.MAX_RETRIES.defaultValue(),
                         null,
-                        SCHEMA.toPhysicalRowDataType());
+                        SCHEMA.toPhysicalRowDataType(),
+                        FactoryMocks.IDENTIFIER.asSummaryString());
```

Comment:
could we test in some way the source naming?

Reply:
![image](https://github.com/apache/flink-connector-jdbc/assets/64569824/cf2db1b0-a933-40cd-8ea6-b44614c13713)
![image](https://github.com/apache/flink-connector-jdbc/assets/64569824/4a88a206-1390-4f61-aaef-d2e0208c9e51)


I added the corresponding cases for testing.
As shown in the screenshot pics, The name is overwrite by the translation mechanism.
Maybe we should drop the `tableIdentifier` attribute at present ?

Reply:
If is overwrite I would say yes, we should drop the attribute, as at the end is not used..

Reply:
@eskabetxe updated!   :)